src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -147,6 +135,7 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
         if (this.exercise && this.exercise.participations && this.exercise.participations.length > 0) {
             this.exercise.participations.forEach(participation => {
                 this.participationWebsocketService.addParticipation(participation, this.exercise);
+                this.participationWebsocketService.addResultsToParticipation(participation, ...participation.results);
```

Comment:
Here I make sure, that the participation websocket service gets all results for each participation.